Changes

```diff
@@ -14,6 +14,9 @@ be mentioned in the 4.06 section below instead of here.)
   (Vincent Laviron, with help from Pierre Chambart,
    reviews by Gabriel Scherer and Luc Maranget)
```

Comment:
N.B. that change entry was in a "Code generation and optimizations" before #1370 got merged, but then somehow appeared in "Type system", I assume that's due to a merge/rebase mistake.